doc/adr/update-toc

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Usage: ./update-toc.sh [target=index.md] [docs_pattern=0*.md] [marker=toc]
```

Comment:
definitely a great improvement; now we just need to remember to run this script!

Reply:
"Dreams"